data/scripts.js

```diff
@@ -184,7 +184,7 @@ let BattleScripts = {
 
 		let movename = move.name;
 		if (move.id === 'hiddenpower') movename = 'Hidden Power';
-		if (sourceEffect) attrs += '|[from]' + this.getEffect(sourceEffect);
+		if (sourceEffect) attrs += '|[from] ' + sourceEffect.fullname;
```

Comment:
I've been trying to remove all the spaces from the protocol, actually.

Reply:
```suggestion
		if (sourceEffect) attrs += `|[from]${sourceEffect.fullname}`;
```

Reply:
I merged this in as I was rebasing anyway.